examples/00-load/create-sphere.py

```diff
@@ -0,0 +1,239 @@
+"""
+.. _create_sphere:
+
+Create Sphere Mesh Multiple Ways
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to create meshes in different ways.
+
+"""
+# sphinx_gallery_thumbnail_number = 5
+
+# %%
+from __future__ import annotations
+
+import numpy as np
+
+import pyvista as pv
+
+# %%
+# Simple Sphere
+# ~~~~~~~~~~~~~
+# The quickest method to get a Sphere mesh is to use :func:`pyvista.Sphere`.
+
+mesh = pv.Sphere()
+mesh.plot(show_edges=True)
+
+# %%
+# This gives an :class:`pyvista.PolyData` mesh, i.e. a 2D surface. In this case, it is watertight
+# and encloses a volume.
+
+mesh
+
+# %%
+# The cells are triangles. For example, the first cell
+
+mesh.get_cell(0).type
+
+# %%
+# Structured quadrilateral mesh of Sphere
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The structure of the mesh can be important. Instead of a
+# triangulated mesh, it can be useful to have a structured
+# mesh that has an i-j-k ordering that allows for simplified
+# cell connectivity.
+#
+# The points are generated as a regular grid in spherical coordinates.
+# Here, we will used the convention that ``theta`` is the
+# azimuthal angle, similar to longitude on the globe.  ``phi`` is the
+# polar angle, similar to latitude on the globe.
+
+radius = 0.5
+ntheta = 9
+nphi = 12
+theta = np.linspace(0, 2 * np.pi, ntheta)
+phi = np.linspace(0, np.pi, nphi)
+
+r_, phi_, theta_ = np.meshgrid([radius], phi, theta, indexing='ij')
+x, y, z = pv.spherical_to_cartesian(r_, phi_, theta_)
+mesh = pv.StructuredGrid(x, y, z)
+
+# %%
+# The mesh has quadrilateral cells. The cells that look triangular
+# at the poles are actually degenerate quadrilaterals, i.e. two
+# points are coincident at the pole, as will be shown later.
+
+mesh.plot(show_edges=True)
+
+# %%
+# The mesh is of type :class:`pyvista.StructuredGrid`.
+
+mesh
+
+# %%
+# The first cell is at the top pole, and it is a Quadrilateral.
```

Comment:
Maybe add the quad cross-ref here instead of above (or keep both?)
```suggestion
# The first cell is at the top pole, and it is a :attr:`~pyvista.CellType.QUAD` cell.
```